masonite/providers/UploadProvider.py

```diff
@@ -1,4 +1,4 @@
-""" A Mail Service Provider """
+""" A Upload Service Provider """
```

Comment:
This should be `""" An Upload Service Provider """` and not `""" A Upload Service Provider """`

Reply:
Ready, check please :)